docs/conf.py

```diff
@@ -138,29 +142,33 @@ def fix_circleci(x): return None
 # -- Options for the Sphinx gallery -------------------------------------------
 if has_sphinx_gallery:
     from sphinx_gallery.sorting import ExampleTitleSortKey
+
     extensions += ["sphinx_gallery.gen_gallery"]
     path = Path.cwd()
-    example_dir = path.parent.joinpath('examples')
+    example_dir = path.parent.joinpath("examples")
     sphinx_gallery_conf = {
-        'backreferences_dir': str(path.joinpath('generated', 'modules')),
-        'filename_pattern': '^((?!skip_).)*$',
-        'examples_dirs': example_dir,
-        'within_subsection_order': ExampleTitleSortKey,
-        'gallery_dirs': path.joinpath('generated', 'gallery'),
-        'default_thumb_file': path.joinpath('logo', 'sunpy_icon_128x128.png'),
-        'abort_on_example_error': False,
-        'plot_gallery': True,
-        'doc_module': ('sunpy')
+        "backreferences_dir": str(path.joinpath("generated", "modules")),
```

Comment:
you are just causing chaos when we retemplate this now...

Reply:
Black your code.